sdk/signalr/azure-messaging-webpubsubservice/README.md

```diff
@@ -0,0 +1,148 @@
+>>> response = client.send_request(request)
+>>> response
+<RequestsTransportResponse: 202 Accepted>
+>>> response.status_code 
+202
+>>> with open('file.json', 'r') as f:
+>>>    request = build_send_to_all_request('ahub', content=f, content_type='application/json')
+>>>    response = client.send_request(request)
+>>> print(response)
+<RequestsTransportResponse: 202 Accepted>
+```
+
+## Key concepts
+
+### Hub
+
+Hub is a logical set of connections. All connections to Web PubSub connect to a specific hub. Messages that are broadcast to the hub are dispatched to all connections to that hub. For example, hub can be used for different applications, different applications can share one Azure Web PubSub service by using different hub names.
+
+### Group
+
+Group allow broadcast messages to a subset of connections to the hub. You can add and remove users and connections as needed. A client can join multiple groups, and a group can contain multiple clients.
+
+### User
+
+Connections to Web PubSub can belong to one user. A user might have multiple connections, for example when a single user is connected across multiple devices or multiple browser tabs.
+
+### Connection
+
+Connections, represented by a connection id, represent an individual websocket connection to the Web PubSub service. Connection id is always unique.
+
+### Message
+
+A message is either a UTF-8 encoded string, json or raw binary data.
+
+## Troubleshooting
+
+### Logging
+
+This SDK uses Python standard logging library.
+You can configure logging print out debugging information to the stdout or anywhere you want.
+
+```python
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+````
+
+Http request and response details are printed to stdout with this logging config.
+
+## Next steps
+
+More examples are coming soon...
+
+## Contributing
+
+This project welcomes contributions and suggestions. Most contributions require
+you to agree to a Contributor License Agreement (CLA) declaring that you have
+the right to, and actually do, grant us the rights to use your contribution.
+For details, visit https://cla.microsoft.com.
+
+When you submit a pull request, a CLA-bot will automatically determine whether
+you need to provide a CLA and decorate the PR appropriately (e.g., label,
+comment). Simply follow the instructions provided by the bot. You will only
+need to do this once across all repos using our CLA.
+
+This project has adopted the
+[Microsoft Open Source Code of Conduct][code_of_conduct]. For more information,
+see the Code of Conduct FAQ or contact opencode@microsoft.com with any
+additional questions or comments.
+
+<!-- LINKS -->
+[webpubsubservice_docs]: https://aka.ms/awps/doc
+[azure_cli]: https://docs.microsoft.com/cli/azure
+[azure_sub]: https://azure.microsoft.com/free/
+[webpubsubservice_client_class]: https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/signalr/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/__init__.py
```

Comment:
move the `azure-messaging-webpubsubservice` into a new `webpubsub` folder instead of using `signalr`?